modules/guides/examples/javascript-udfs/add-airline-with-date-response.json5

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "airline": {
+      "ca1": "Q5",
+      "callsign": "RAZ-AIR",
+      "country": "United Kingdom",
+      "icao": "MLA",
+      "id": 1220,
+      "name": "Raz Air",
+      "opdate": "2022-02-25T09:40:57.589Z",    // <.>
```

Comment:
I've noticed that you're using the json5 file extension, presumably so that you can include comments in the JSON for asciidoc tagging. But you're not actually using any of the other JSON5 features. Have you considered using jsonc (JSON with comments) as a file extension, for consistency with other example files?

Reply:
Yep, can certainly do that.